### DISCUSSION TITLE
did renovatebot stop working?

### DISCUSSION THREAD

**Discussion by electriquo** [renovatebot/renovate#11161] [Request Help]
i am using renovatebot on all my repositories, where some of my repositories have the same dependencies and have the same renovatebot configuration.

in the past, renovatebot used to open a pull-request on all of my repositories around the same time and i loved it.
few days ago, renovatebot opened a pull-request with dependency updates only on 1 of my repositories.

1. did something happen to renovatebot? if not, when to expect for a pr from renovatebot about dependencies update?
2. how can i run renovatebot on demand?

**Comment by viceice** [comment]
What type of the bot are you using? GitHub app or self-hosted? What do the debug logs say?

**Reply by electriquo** [reply]
I am using the Github App

**Reply by viceice** [reply]
https://app.renovatebot.com

**Comment by rarkins** [comment]
Check app.renovatebot.com for logs

**Reply by electriquo** [reply]
Thanks.
Looking at the logs did not reveal any error or issue.

I wonder why it works on one repository and does not work on the other when the same renovatebot configuration is used.

Would it help if I will share with your renovate bot run id?

**Reply by rarkins** [reply]
No, because we don't have time to look through everyone's entire logs. You need to narrow down your description from
"Stopped working" to something more specific and then identify the relevant logs yourself and ask here only after then if any are confusing.

**Reply by electriquo** [reply]
I use renovatebot github app (25.68.0) with [regex manager](https://docs.renovatebot.com/modules/manager/regex/) with pypi [datasource](https://docs.renovatebot.com/modules/datasource/) to look for an open-source package update.

Looking at the trace logs, I do see that renovatebot reads the configuration, but it seems like renovatebot does not look for the package update on pypi.

In the full stats/allRequests there is a list of GET requests and it lists Github API, DockerHub and more but not pypi.

```shell
{
  "hostStats": [
    "api.github.com, 6 requests, 325ms request average, 0ms queue average",
    "auth.docker.io, 3 requests, 281ms request average, 4ms queue average",
    "index.docker.io, 8 requests, 333ms request average, 0ms queue average",
    "repo.maven.apache.org, 16 requests, 244ms request average, 0ms queue average"
  ],
  "totalRequests": 33
}
```

Anything else I can share with you to get your hint and lead?

And **thanks again** for the responsiveness.

**Reply by rarkins** [reply]
Look for the "packageFiles with updates" log message to see the list of detected files and dependencies

**Reply by electriquo** [reply]
As said, I have the same renovate configurations on many repositories, but renovate opened a pull-request on a single repository.

The functionality works in general (relates #11166, where the [results can be seen here](https://github.com/foolioo/renovate-pre-commit/pulls)). But when looking at the "packageFiles with updates" over the trace log of repository which the functionality did not work, I do not see pypi in the datasourceTemplate (inside the "regex" array) :(

**Reply by rarkins** [reply]
Assuming the logs confirm that Renovate is running on the other repos, and you have identical config, then it almost definitely means that there's something about the other repos which is different which right now you are overlooking or dismissing as unimportant.

Therefore either your config isn't identical, or your repos aren't as similar as you think.

**Reply by electriquo** [reply]
Found the issue, someone added a [zero-width space](https://en.wikipedia.org/wiki/Zero-width_space) to the file which is being parsed.

When this character (zero-width space) was removed, renovate was happy.

This discussion can be closed.

# @rarkins: THANK YOU FOR YOUR HELP! APPRECIATED!
